CHANGELOG.md

```diff
@@ -4,10 +4,12 @@
 
 v2.4.1 is a maintenance release with the following fixes and enhancements:
```

Comment:
Version can be changed to 2.5.0  when we merge `delete_records`

Reply:
Already done that in delete_records as that is a new feature.